src/documenti/interni/NdP/contenuti/processi_supporto.tex

```diff
@@ -177,12 +177,12 @@ \subsection{Consuntivo}
 \begin{itemize}
     \item Consuntivo orario:
         \begin{itemize}
-            \item Tabella con la struttura identica a quella del preventivo orario dove si indica tra parentesi tonde la differenza tra le ore consuntivate e quelle preventivate;
+            \item Tabella con la struttura identica a quella del preventivo orario dove si indicano le ore consuntivate e tra parentesi tonde la differenza tra le ore consuntivate e quelle preventivate;
             \item Tabella con la struttura identica a quella del preventivo orario dove si indicano le ore rimaste per ruolo e per persona.
         \end{itemize}
     \item Consuntivo economico:
         \begin{itemize}
-            \item Tabella con la struttura identica a quella del preventivo economico dove si indica tra parentesi tonde la differenza tra ore consuntivate e preventivate, con la conseguente differenza, sempre tra parentesi tonde, tra i costi consuntivati e quelli preventivati.
+            \item Tabella con la struttura identica a quella del preventivo economico dove si indicano in una colonna le ore consuntivate e tra parentesi tonde la differenza tra ore consuntivate e preventivate, e nell'altra colonna i costi consuntivati e con la corrispondente differenza, sempre tra parentesi tonde, tra i costi consuntivati e quelli preventivati.
```

Comment:
c'è ancora quel "e con" in cui toglierei il "con" perché leggendo la frase stona. Comunque queste piccole modifiche sono talmente insulse che avrebbe senso che le cambiassi io e siamo apposto, invece di scambiarci mille messaggi per sta roba, ma ormai fallo tu e siamo apposto